kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm-types.md

```diff
@@ -323,7 +323,7 @@ A cons-list is used for the EVM wordstack.
     syntax Bytes ::= "#write" "(" Bytes "," Int "," Int ")" [function]
                    | Bytes "[" Int ":=" Bytes "]" [function, total, klabel(mapWriteRange)]
  // --------------------------------------------------------------------------------------
-    rule #write(WM, IDX, VAL) => padRightBytes(WM, IDX +Int 1, 0) [ IDX <- VAL ]
+    rule #write(WM, IDX, VAL) => padRightBytes(WM, IDX +Int 1, 0) [ IDX <- VAL ] [concrete]
```

Comment:
Why not just have `rule #write(WM, IDX, VAL) => WM [ IDX := #buf(1, VAL) ]` here and not use the one above?

Reply:
Understood. Then could the simplification in `buf.md` be unconditional? Would that be enough to never see `padRightBytes` in the symbolic world?

Reply:
Yes it could reasonably be unconditional.

Reply:
Have to test with the use-case that inspired this, and see if it works.